markdown.go

```diff
@@ -1082,6 +1089,21 @@ func expandTabs(out *bytes.Buffer, line []byte, tabSize int) {
 	}
 }
 
+// Extract indent finds the appropriate indentation for the next block
+// where line can be inside of a list.
+func extractIndent(line []byte, tabSize int) []byte {
```

Comment:
Sorry, missed the last part... I did not notice anything that exactly... although I may have missed it.